demo/repo/Jenkinsfile

```diff
@@ -17,26 +17,31 @@ parallel(longerTests: {
     runTests(servers, 20)
 })
 
-stage name: 'Staging', concurrency: 1
-node {
-    servers.deploy 'staging'
-}
+milestone ordinal: 1
+lock (resource: 'staging-server', inversePrecedence: true) {
```

Comment:
For consistency with other step usages (and _Snippet Generator_ output), please elide the space before the opening parenthesis. See for example `dir` above.